Shared/sdk/SharedUtil.SysInfo.h

```diff
@@ -8,9 +8,12 @@
  *  Multi Theft Auto is available from http://www.multitheftauto.com/
  *
  *****************************************************************************/
+#pragma once
 
 #ifdef WIN32
-
+#include <vector>
+#include "SString.h"
+#include <Windows.h>
```

Comment:
```suggestion
#include <windows.h>
```
For cross-compilation

Reply:
The file is called `Windows.h` (uppercase `W`) in the winsdk, so I'm not sure why we'd need to lowercase it for cross compiling.

Reply:
It only matters if you compile on a system with case-sensitive paths and the Windows header is actually lowercase (like for MinGW)

Reply:
Interesting, haven't run into that yet